client/task-list/tasks/payments/components/PaymentMethodList/PaymentMethodList.js

```diff
@@ -9,15 +9,16 @@ import {
 	CardMedia,
 	CardHeader,
 	CardDivider,
+	__experimentalText as Text,
```

Comment:
Let's update this to import from `@woocommerce/experimental`.

Reply:
Done! I also replaced it in a few of the other components in this feature.